[DEFECT] Docker Maven Plugin fails when running `test-integration-ide` command

### Problem Statement

Found when working on fix for:

* #34357

In a local environment, executing the `test-integration-ide` command from our `justfile` is throwing an error in the terminal. This is keeping developers from easily running and debugging Integration Tests from the IDE.

### Steps to Reproduce

* Set up a local dotCMS development environment from the `main` branch.
* First, compile dotCMS as usual.
* Execute the `test-integration-stop` command from the `justfile`.
* You'll get the following error in the terminal:
```bash
[DB]PostgreSQL init process complete; ready for start up.
[DB]
[DB]2026-01-27 22:19:58.458 UTC [1] LOG:  starting PostgreSQL 15.4 (Debian 15.4-2.pgdg120+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
[DB]2026-01-27 22:19:58.458 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
[DB]2026-01-27 22:19:58.458 UTC [1] LOG:  listening on IPv6 address "::", port 5432
[DB]2026-01-27 22:19:58.460 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
[DB]2026-01-27 22:19:58.462 UTC [67] LOG:  database system was shut down at 2026-01-27 22:19:58 UTC
[DB]2026-01-27 22:19:58.465 UTC [1] LOG:  database system is ready to accept connections
[INFO] DOCKER> [ankane/pgvector:latest] "database": Stop and removed container 4f6f6eca7f10 after 0 ms
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  37.936 s
[INFO] Finished at: 2026-01-27T16:19:58-06:00
[INFO] ------------------------------------------------------------------------
[WARNING] 
[WARNING] Plugin validation issues were detected in 1 plugin(s)
[WARNING] 
[WARNING]  * io.fabric8:docker-maven-plugin:0.43.4
[WARNING] 
[WARNING] For more or less details, use 'maven.plugin.validation' property with one of the values (case insensitive): [BRIEF, DEFAULT, VERBOSE]
[WARNING] 
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.43.4:start (docker-start) on project dotcms-integration: Execution docker-start of goal io.fabric8:docker-maven-plugin:0.43.4:start failed: Cannot invoke "com.google.gson.JsonElement.isJsonNull()" because the return value of "com.google.gson.JsonObject.get(String)" is null -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
error: Recipe `test-integration-ide` failed on line 155 with exit code 1
```

### Acceptance Criteria

- [ ] Setting up your local environment with the updated library must not cause any issues.

### dotCMS Version

Latest main

### Severity

High - Major functionality broken

### Links

NA

--- a/.github/actions/core-cicd/maven-job/action.yml
+++ b/.github/actions/core-cicd/maven-job/action.yml
@@ -238,6 +238,20 @@ runs:
         name: docker-build-context
         path: dotCMS/target/docker-build.tar
 
+    - id: get-sdkman-version
+      name: Get SDKMAN Java Version
+      if: ${{ inputs.generate-docker == 'true' }}
+      shell: bash
+      run: |
+        if [ -f ".sdkmanrc" ]; then
+          SDKMAN_JAVA_VERSION=$(awk -F= '/^java=/ {print $2}' .sdkmanrc)
+          echo "SDKMAN_JAVA_VERSION=${SDKMAN_JAVA_VERSION}" >> $GITHUB_ENV
+          echo "Extracted SDKMAN_JAVA_VERSION: ${SDKMAN_JAVA_VERSION}"
+        else
+          echo "Error: .sdkmanrc file not found"
+          exit 1
+        fi
+
     - id: build-docker-image-from-archive
       name: Build Docker Image from Archive
       if: ${{ inputs.generate-docker == 'true' }}
@@ -248,7 +262,7 @@ runs:
         tar -xf dotCMS/target/docker-build.tar -C /tmp/docker-context
 
         # Build the Docker image from the extracted context
-        docker build -t dotcms/dotcms-test:${{ inputs.version }} /tmp/docker-context
+        docker build --build-arg SDKMAN_JAVA_VERSION="${SDKMAN_JAVA_VERSION}" -t dotcms/dotcms-test:${{ inputs.version }} /tmp/docker-context
 
         # Verify the image was created
         docker images dotcms/dotcms-test:${{ inputs.version }}

--- a/.github/actions/core-cicd/maven-job/action.yml
+++ b/.github/actions/core-cicd/maven-job/action.yml
@@ -238,6 +238,21 @@ runs:
         name: docker-build-context
         path: dotCMS/target/docker-build.tar
 
+    - id: build-docker-image-from-archive
+      name: Build Docker Image from Archive
+      if: ${{ inputs.generate-docker == 'true' }}
+      shell: bash
+      run: |
+        # Extract the build context from the tar archive
+        mkdir -p /tmp/docker-context
+        tar -xf dotCMS/target/docker-build.tar -C /tmp/docker-context
+
+        # Build the Docker image from the extracted context
+        docker build -t dotcms/dotcms-test:${{ inputs.version }} /tmp/docker-context
+
+        # Verify the image was created
+        docker images dotcms/dotcms-test:${{ inputs.version }}
+
     - id: save-docker-image
       name: Save Docker Image to Tar File
       if: ${{ inputs.generate-docker == 'true' }}

